facebook/Makefile

```
@@ -178,18 +197,18 @@ run-R: $(CIDS) $(CIDS_EXP)
 	grep "scheduled core" tmp ; \
 	[ "$$?" -eq 1 ]
 
-pipeline: scratch init-qualtrics params.json $(WEIGHTS) run-R post-cids post-experimental-cids post-individual post-individual-raceeth post-done tidy
+pipeline: scratch init-qualtrics params.json $(WEIGHTS) run-R post-cids post-experimental-cids post-individual post-individual-raceeth post-done tidy $(WEEKLY_WEIGHTS)
 	grep $(TODAY) params.json
 	[ -f $(YESTERDAY) ] && rm $(YESTERDAY) || true
 	touch $@
 	echo "SUCCESS: $(DRY_MESSAGE)pipeline complete" >> $(MESSAGES)
 	chmod o+w $(MESSAGES)
 
-weekly-weights-pipeline: scratch init-qualtrics params.weekly-weights.json $(WEEKLY_WEIGHTS) run-R post-individual post-individual-raceeth tidy-weekly-weights
-	# Want to schedule on Wednesday or Thursday to be very likely to have received weekly weights.
+weekly-weights-pipeline: scratch init-qualtrics params.weekly-weights.json run-R post-individual post-individual-raceeth tidy-weekly-weights
 	touch $@
-	echo "SUCCESS: $(DRY_MESSAGE)weekly weights pipeline complete" >> $(WEEKLY_WEIGHTS_MESSAGES)
-	chmod o+w $(WEEKLY_WEIGHTS_MESSAGES)
+	echo "SUCCESS: $(DRY_MESSAGE)weekly weights pipeline complete" >> $(MESSAGES)
+	chmod o+w $(MESSAGES)
```

avoid collision with a check in `cron.sh` that controls whether the covidcast CSVs are posted to the server:
```suggestion
	echo "SUCCESS: $(DRY_MESSAGE)completed weekly weights pipeline" >> $(MESSAGES)
```